server/lib/mod-utils.js

```diff
@@ -145,6 +145,7 @@
           },
       });
     }
+    const queryStatus = (tab === 'bureau') ?'responded' : 'responded,panel,juror';
```

Comment:
can just be;
`const queryStatus = tab === 'bureau' ? 'responded' : 'responded,panel,juror';`